src/borg/archiver.py

```diff
@@ -2604,6 +2604,22 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
                 group.add_argument('--last', metavar='N', dest='last', default=0, type=positive_int_validator,
                                    help='consider last N archives after other filters were applied')
 
+        def define_borg_mount(subparser):
```

Comment:
guess this should be `parser`.

that the parser can be in fact a subparser sometimes is a detail not needed here (and also is not true for borgfs).